hw/hw11/mailroom.py

```diff
@@ -0,0 +1,120 @@
+if __name__ == '__main__':
+
+    donors = {
+        'Yoo Hoo': [3000000, 13438902, 2532],
+        'Elon Musk': [3, 6, 99],
+        'Ronald Mcdonald': [200, 448, 500]
+        }
+
+    def start():
+        menu = ''' Choose from the following:\n
+            T - Send a (T)hank You\n
+            R - Create a (R)eport\n
+            Quit - Quit the program\n'''
+
+        print('Welcome to Mailroom Madness')
+
+        answer = input(menu)
+        valid_input(answer)
+
+        while answer:
+            answer = input('Please enter a valid selection:\n' + menu)
+            answer = valid_input(answer)
+
+    def valid_input(answer):
+        bad_input = True
+
+        try:
+            answer = answer.lower()
+            if answer == 'quit':
+                exit()
+            elif answer == 't':
+                thanks()
+            elif answer == 'r':
+                report()
+            else:
+                return bad_input
+        except TypeError:
+                return bad_input
```

Comment:
Watch your whitespece. This should be 4 spaces only. Make sure your linter is functioning.